### DISCUSSION TITLE
Home Mode

### DISCUSSION THREAD

**Discussion by EItanMao** [djdd87/SynoAI#121] [Ideas]
Is it possible to have the option to send notifications (preferably per notification type i.e. Telegram, webhook etc..) based on Surveillance Station's Home Mode (Home or Away)?

Reason for this is in my use case that I do want the webhook notifications to work 24/7 since they alert my smart home system (SmartThings) on motion, but I would like to configure Telegram notifications only when I am away from home.

Best Eitan.

**Comment by djdd87** [comment]
Just configure this in Synology Surveillance Station from the "Home Mode" configuration:

Enable setting of Action Rules
![image](https://user-images.githubusercontent.com/22151993/160615432-e1b2c9e6-7be9-4322-bdc8-9b0694427f01.png)

Only enable the action rules you want to run:
![image](https://user-images.githubusercontent.com/22151993/160615572-308a943f-c660-47df-9894-56da162fe5cf.png)

**Reply by EItanMao** [reply]
Thx for this wasent aware of the option worked for me but had to spin up another instance to support the 24/7 notifications and the ones that trigger only when I am away.

And by the way great peace of work thank you!!!

**Comment by mostlyflawles** [comment]
I don't get how to configure this, i've enabled "Enable the setting of action rule" and regardless if home mode is enabled or disabled the action is triggered. Are there any other steps I need to do?

I've also played with this option, but I assume is only an event trigger when the mode shifts. Because no notification are triggered at all with this setting.
![Leave-Enter](https://github.com/djdd87/SynoAI/assets/25802524/6bb27f6a-ec18-45cf-af51-26b6ee155b46)

**Reply by djdd87** [reply]
You shouldn't need to set the leave/enter home mode event in here as that's irrelevant. 

What does your home mode action rule list look like (as per my screemshot above)?

**Comment by mostlyflawles** [comment]
It works now, I think my logical went wrong regards to "Enable" in the Action rule list.

"not" Enable = Will be triggered when not home

![image](https://github.com/djdd87/SynoAI/assets/25802524/52680daf-235f-482a-a7a4-3880aff6d8a3)
